RELEASE.md

```diff
@@ -0,0 +1,63 @@
+# lnc-core Release Process
+
+This document describes the steps needed to release a new version of
+`@lightninglabs/lnc-core` and publish the package to the NPM registry.
+
+The steps below for Bumping proto versions and Versioning should be done in
+a PR with proper review.
+
+## Bumping proto versions
+
+When new versions of LND, Loop, Pool, Faraday, or Lightning Terminal are
+released, we should update the generated types in this package to match the
+latest versions.
+
+To do this, you need to update the `config` section in the
```

Comment:
It would be nice if we had a manifest somewhere with all the latest releases, and we could update this programmatically 😄

Reply:
Not a bad idea. We could parse the LNC `go.mod` file to get the specific versions we should be using here.